webhook/handlers/UserController.js

```diff
@@ -0,0 +1,177 @@
+const db = require('../../models');
+
+class UserRegistrationController {
+  
+  async newUserIntent(agent, body) {
+    console.log("Storing new User in DB");
+    let fName = agent.parameters["user-given-name"];
+    let cNum = body.From;
+
+    let usercheck = await db.userWA.findAll({where: {wa_phone_number: cNum}});
+    if(usercheck.length > 0) return "Phone Number already Exists";
+    fulfillmentText = "Hi " + fName + ", welcome to E-Medic. Please check your mail for a confirmation email from us.\
+            \nMeanwhile please share your location in the chat. Clip Icon -> Location";
+    db.userWA.create({
+            "first_name": fName,
+            "last_name": agent.parameters["last-name"],
+            "email": agent.parameters["email"],
+            "wa_phone_number": cNum
+        })
+        .then(() => {
+            return fulfillmentText;
+        })
+        .catch((err) => {return err;});
+  }
+
+  addLocation(body){    
+    let fulfillmentText = "Updated Location";
+    db.userWA.update({
+      "loc_long": body.Logitude.toSring(),
+      "loc_lat": body.Latitude.toSring()
+    }, { where: {"wa_phone_number": body.From}})
+    .then(() => {
+      return fulfillmentText;
+    })
+    .catch((err) => {return err;});
+  }
```

Comment:
I didn't understand what changes you need to this function?

Reply:
Look at the comment above. 
We should ask `user` for full address (text), instead of location (I discussed today with Sreelatha). 
We will use the clip -> location only just before booking consultation.
Remove this function from the user reg workflow.

Reply:
Ok yeah. We can use this function for anytime the user sends location. Don't need to call it in the beginning.